src/image/imageengine.js

```diff
@@ -70,6 +85,12 @@ export default class ImageEngine extends Plugin {
 			.consuming( { attribute: [ 'alt' ] } )
 			.toAttribute( viewImage => ( { key: 'alt', value: viewImage.getAttribute( 'alt' ) } ) );
 
+		// Build converter for srcset attribute.
+		buildViewConverter().for( data.viewToModel )
+			.from( { name: 'img', attribute: { srcset: /./ } } )
```

Comment:
This object is passed down to the `ViewMatcher`. Matcher will test this regexp against all `img` elements with `srcset` attribute, so it will be true for any non-empty attribute. 

It looks odd to me too to be honest and I needed to look inside `ViewMatcher` to be sure. Maybe matcher could accept also an array of attributes which will mean "match if attribute is present".